docs/index.rst

```diff
@@ -2,74 +2,162 @@
 Dask-Jobqueue
 =============
 
-*Easy deployment of Dask Distributed on job queuing systems like
-PBS, Slurm, and SGE.*
+*Easily deploy Dask on job queuing systems like PBS, Slurm, MOAB, and SGE.*
 
-Motivation
-----------
 
-1. While ``dask.distributed`` offers a flexible library for distributed computing
-   in Python, it is not always easy to deploy on systems that use job queuing
-   systems. Dask-jobqueue provides a Pythonic interface for deploying and
-   managing Dask clusters.
-2. In practice, deploying distributed requires customization, both for the
-   machine(s) that it will deployed on and for the specific application it will
-   be deployed for. Dask-jobqueue provides users with an intuitive interface for
-   customizing dask clusters.
+The Dask-jobqueue project makes it easy to deploy Dask on common job queuing
+systems typically found in high performance supercomputers, academic research
+institutions, and other clusters.  It provides a convenient interface that is
+accessible from interactive systems like Jupyter notebooks, or batch jobs.
 
 Example
 -------
 
 .. code-block:: python
 
    from dask_jobqueue import PBSCluster
+   cluster = PBSCluster()
+   cluster.scale(10)         # Ask for ten worker nodes
 
-   cluster = PBSCluster(processes=6, threads=4, memory="16GB")
-   cluster.start_workers(10)
+   # wait for jobs to arrive, depending on the queue, this may take some time
 
    from dask.distributed import Client
-   client = Client(cluster)
+   client = Client(cluster)  # Connect Dask to the worker nodes
```

Comment:
Here again I take it the comment is a simplification that you believe is more appropriate for common users?

Reply:
Yes

Reply:
Alternatives welcome